foxy/distribution.yaml

```diff
@@ -1918,6 +1918,12 @@ repositories:
       url: https://github.com/ros2/message_filters.git
       version: foxy
     status: maintained
+  micro_ros_msgs:
+    release:
```

Comment:
could you add source and doc entries?
it's not a requirement, but it's usually done

Reply:
Should I add it manually here or do I have to redo it using the bloom command?

Sorry, but it is our first time releasing with bloom and I have some doubts. One of them is: is normal to release the same version (1.0.0 in this case) for different ROS distributions (Rolling, Foxy, and Galactic)?

Reply:
> Should I add it manually here or do I have to redo it using the bloom command?

Both ways are fine.

> One of them is: is normal to release the same version (1.0.0 in this case) for different ROS distributions (Rolling, Foxy, and Galactic)?

If your code doesn't change between the distributions, then yes, it's normal to use the same version.

Reply:
Done. One more question: I'm expecting that this generates a binary installable via `apt` that's right?

Then, if I have this package as a dependency of another will the `apt` package manager solve the interdependency automatically?

Reply:
> Done. One more question: I'm expecting that this generates a binary installable via apt that's right?

Yes, this generates binaries that in ubuntu can be installed via `apt` (you have to add the ros2 apt repository to have them available).

> Then, if I have this package as a dependency of another will the apt package manager solve the interdependency automatically?

I'm not sure if this is what you're asking, but when you install the package it will also install all its required dependencies.

Maybe related, but you can also use rosdep to install dependencies of the packages in your workspace, see http://wiki.ros.org/rosdep#Install_dependency_of_all_packages_in_the_workspace.

---

The binary won't be available until next sync though, syncs roughly happen each 15 days (sometimes it can take a month).